src/test/resources/TCS.feature

```diff
@@ -3,11 +3,13 @@ Feature: Navigation bar
     Without logging in
     I can the login workflow
 
+    @test
     Scenario: I can try to login using a correct and wrong password.
         Given I navigate to www.saucedemo.com
         When Login using a correct password
```

Comment:
El when puede ser mas descriptivo y asegurarse que solo se encargue de loguear al usuario. Por ejemplo  "When standard user logs in with valid credentials"